tests/cli/main.cpp

```diff
@@ -51,18 +51,49 @@
  * Helper Methods
  *********************/
 
-// hack:  import create_example_genesis() even though it's a way, way
-// specific internal detail
+/////////
```

Comment:
I'm not sure whether Doxygen recognize this format. It does recognize `/** */`.

Reply:
Yes, it is recognized. https://www.stack.nl/~dimitri/doxygen/manual/docblocks.html

I prefer /* */, but some like // as it allows you to comment out big chunks of code.

Reply:
According to the link, maybe need 3 slashes?
```
/////////////
/// doc
///////////
```

Reply:
Yes, you are correct. I will modify.